File: src/STORE.Entities/Models/Articulo.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace STORE.Entities.Models
{
    [Table("Articulo")]
    public class Articulo
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ArticuloID { get; set; }

        [StringLength(100)]
        public string Codigo { get; set; }

        [StringLength(250)]
        public string Descripcion { get; set; }
        public decimal Precio { get; set; }
        public string Imagen { get; set; }
        public int Stock { get; set; }
        public List<ArticuloTienda> ArticulosTiendas { get; set; }
        public List<ClienteArticulo> ClientesArticulos { get; set; }
    }
}

File: src/STORE.Front/ClientApp/src/app/articulo/interface/articulo.ts
export interface Articulo {
    articuloTiendaID: number;
    articuloID: number;
    codigo: string;
    descripcion: string;
    precio: number;
    imagen: string;
    stock: number;
    tiendaID: number;
    sucursal: string;
    direccion: string; 
    fecha: Date;
    cantidadCarrito: number;
}